source/patchcanvas.py

```diff
@@ -1212,6 +1214,8 @@ def __init__(self, parent, view):
         self.m_ctrl_down = False
         self.m_mouse_down_init = False
         self.m_mouse_rubberband = False
+        self.m_mouse2_down = False
```

Comment:
This attribute name is a bit ambiguous, don't you think? Are mouse buttons counted from zero or one?

Reply:
Just looked to docs about Qt::MouseButtons - indeed, that use special names for first several buttons, while numbered names refer to extra buttons, probably on gamer mice.
But still i don't remember cases, where numbering begins from 0.
For example, in gtk and some x11 apps (remembering configuring fluxbox) it begins with 1.
Anyway, i will rename it to match to qt docs.

Reply:
Completely missed misleading MButton3 :/